src/test/java/com/iota/iri/conf/ConfigTest.java

```diff
@@ -27,6 +13,25 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.hamcrest.CoreMatchers;
+import org.hamcrest.core.IsCollectionContaining;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.iota.iri.model.HashFactory;
+import com.iota.iri.utils.IotaUtils;
+
```

Comment:
Just making sure, are you using `format.importorder` and `format_settings.epf`?
I get different results on my IntelliJ

Also, I remind you that this should technically be on separate PRs...
I know... it got in accidently with auto-formatting